install.js

```diff
@@ -0,0 +1,52 @@
+const { execSync, spawn, spawnSync } = require('child_process');
+const { existsSync } = require('fs');
+const readline = require('readline');
+
+const isVersionGreaterThan = (version, targetMajor) => Number(version.split('.')[0]) >= targetMajor;
+
+const trimV = (version) => version.startsWith('v') ? version.slice(1) : version;
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const directory = 'logto';
+
+(async () => {
+  if (existsSync(directory)) {
+    throw new Error(`\`${directory}\` already exists in current directory.`);
+  }
+
+  const nodeVersion = execSync('node -v', { encoding: 'utf-8' });
+  const pgOutput = execSync('postgres --version', { encoding: 'utf-8' });
```

Comment:
Could it throw command not found error if non is installed?

Reply:
<img width="273" alt="image" src="https://user-images.githubusercontent.com/14722250/165453238-8838ee59-a78e-4e2b-a4af-80ae3710e5e7.png">
yes it will